src/main/java/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -252,9 +258,9 @@ public IDelegateBridge getBattleBridge() {
   }
 
   /**
-   * Add bombardment units to battles.
+   * Add bombardment units to battles. Made public for test purposes only
    */
-  private void addBombardmentSources() {
+  public void addBombardmentSources() {
```

Comment:
Package-private is the default accessibility, so you just need to omit the access modifier:

```
void addBombardmentSources() {
```